jangjuseong/routes/likeRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
+const likeController = require('../controllers/likeController');
+
+router.post('/:userId/:postId', likeController.createLike);
```

Comment:
- 좋아요 기능도 로그인이 선행되어야 실행할 수 있기 때문에 미들웨어로 validateToken 함수를 사용하면 좋을 것 같습니다.
- 토큰을 검증하는 미들웨어를 사용할 경우 params로 받고 있는 userId를 삭제해주어야 합니다.

```suggestion
router.post('/:postId', validateToken, likeController.createLike);
```